migration/1646295724658-createTokensTable.ts

```diff
@@ -12,6 +12,7 @@ export class createTokensTable1646295724658 implements MigrationInterface {
           address text COLLATE pg_catalog."default" NOT NULL,
           "networkId" integer NOT NULL,
           decimals integer NOT NULL,
+          "mainnetAddress" text,
```

Comment:
I doubt this will fix the issue on machines has run this migration before. I mean we need to alter table in `migration/1647927483869-fillMainnetAddressForSomeTokens.ts`

Reply:
I altered table in `fillMainnet...` before, 
It fixed our migrations, the `Run migrations` passed in Github Action job